xlive/Blam/Engine/interface/first_person_camera.cpp

```diff
@@ -0,0 +1,84 @@
+#include "stdafx.h"
+#include "first_person_camera.h"
+
+#include "Blam/Cache/TagGroups/unit_definition.hpp"
+#include "Blam/Engine/game/player_control.h"
+#include "Blam/Engine/objects/objects.h"
+#include "Blam/Engine/units/units.h"
+
+#include "H2MOD/Modules/CustomVariantSettings/CustomVariantSettings.h"
+#include "H2MOD/Modules/Shell/Config.h"
+#include "H2MOD/Tags/TagInterface.h"
+#include "Util/Hooks/Hook.h"
+
+#define _USE_MATH_DEFINES
+#include <math.h>
+
+static bool b_showHUD = true;
```

Comment:
we should remove all hungarian style bool prefixes

Reply:
that bool wasn't supposed to be there but changed the name of the actual one